titus-supplementary-component/task-relocation-spring/build.gradle

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+configurations.all {
+    exclude group: 'ch.qos.logback', module: 'logback-classic'
```

Comment:
can you add a comment of why this was necessary? also, is this temporary or long term?

Reply:
Because of conflicts between different logging engines. I will add a comment.